hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -104,29 +107,39 @@
 public class ZookeeperFederationStateStore implements FederationStateStore {
 
   private static final Logger LOG =
-      LoggerFactory.getLogger(ZookeeperFederationStateStore.class);
+          LoggerFactory.getLogger(ZookeeperFederationStateStore.class);
 
   private final static String ROOT_ZNODE_NAME_MEMBERSHIP = "memberships";
   private final static String ROOT_ZNODE_NAME_APPLICATION = "applications";
   private final static String ROOT_ZNODE_NAME_POLICY = "policies";
 
-  /** Interface to Zookeeper. */
+  /**
+   * Interface to Zookeeper.
```

Comment:
The old format is common; I would leave it as is

Reply:
I will fix it.